src/expressions/operator-expr.md

```diff
@@ -79,20 +81,18 @@ let a = && && mut 10;
 let a = & & & & mut 10;
 ```
 
-### Raw address-of operators
+### Raw pointer operators
```

Comment:
I am not sure what is the best thing to call it. Another possible name here might be "Raw borrow expression/operator", to make it clear that this is very similar to reference-creating operations. After all this is in a section titled "borrow operators".

Reply:
"Raw borrow" sounds right to me.  A "raw borrow" creates a "raw reference", which, yes, is also a "raw pointer".

Reply:
All right, I have switched the PR to use "raw borrow operator".